src/components/MigrationDetails/index.jsx

```diff
@@ -0,0 +1,254 @@
+            <div
+              title={TITLES[key]}
+              className={styles[`${prefix}${key.replace("-", "_")}`]}
+              style={{ flex: details[key].length }} key={index}></div>
+          ))}
+      </div>
+    </>
+  );
+}
+
+function Breadcrumbs({ children }) {
+  return (
+    <nav aria-label="breadcrumbs">
+      <ul className="breadcrumbs">
+        <li className="breadcrumbs__item">
+          <a className="breadcrumbs__link" href="/">conda-forge</a>
+        </li>
+        <li className="breadcrumbs__item">
+          <a className="breadcrumbs__link" href="/status">Status</a>
+        </li>
+        <li className="breadcrumbs__item">
+          <a className="breadcrumbs__link" href="/status#migrations">
+            Migrations
+          </a>
+        </li>
+        <li className="breadcrumbs__item breadcrumbs__item--active">
+          <a className="breadcrumbs__link" href="">{children}</a>
+        </li>
+      </ul>
+    </nav>
+  );
+}
+
+function Filters({ counts, filters, onFilter }) {
+  const icon = styles.migration_details_filter_icon;
+  return (
+    <div className={styles.migration_details_filter}>
+      {ORDERED.map(([key, title], index) => {
+        const prefix = "migration_details_filter_";
+        const base = `${prefix}${key.replace("-", "_")}`;
+        return (
+        <div
+          className={[
+            styles.migration_details_filter_button,
+            styles[base],
+            filters[key] && styles[`${base}_on`]
+          ].join(" ")}
+          key={index}
+          onClick={() => onFilter(key)}>
+          {filters[key] ?
+            <i className={`${icon} fa-solid fa-filter-circle-xmark`}></i> :
+            <i className={`${icon} fa-solid fa-filter`}></i>
+          } {title} ({counts[key]})
+        </div>);
+      })}
+    </div>
+  );
+}
+
+function Graph(props) {
+  const [error, setState] = useState("");
+  const url = urls.migrations.graph.replace("<NAME>", props.children);
+  const onError = (error) => setState(error);
+  return (
+    <div>
+      {error ?
+        `Graph is unavailable.` :
+        <SVG onError={onError} src={url} />}
+    </div>
+  );
+}
+
+function Table({ details }) {
+  const [filters, setState] = useState(ORDERED
+      .reduce((filters, [status]) => ({ ...filters, [status]: true }), {}));
+  const feedstock = details._feedstock_status;
+  const rows = ORDERED.reduce((rows, [status]) => (
+    filters[status] ? rows :
+      rows.concat((details[status]).map(name => ([name, status])))
+  ), []);
+  return (
+    <>
+      <Filters
+        counts={ORDERED.reduce((counts, [key]) =>
+          ({ ...counts, [key]: 0 || details[key]?.length }), {})}
+        filters={{ ...filters }}
+        onFilter={key => setState(prev => ({ ...prev, [key]: !prev[key] }))} />
+      {rows.length > 0 && <table>
+        <thead>
+          <tr>
+            <th style={{ width: 200 }}>Name</th>
+            <th style={{ width: 115 }}>Status</th>
```

Comment:
```suggestion
            <th style={{ width: 115 }}>PRs made</th>
```

Reply:
Making a similar change to the migration details page: https://github.com/conda-forge/conda-forge.github.io/pull/2113